spec/consensus/proposer-based-timestamp/pbts-sysmodel_002_draft.md

```diff
@@ -79,172 +86,264 @@ defining a lower bound, a *minimum time* that a correct process assigns to propo
+#### **[PBTS-DERIVED-POL.0]**
 
 If
 
-- there is a valid commit `C` for height `k` and round `r`, and
-- `C` contains a `PRECOMMIT` message from at least one correct process
+- there is a valid `POL(v,r)`, and
+- `POL(v,r)` contains a `PREVOTE` message from at least one correct process,
 
-Then, where `p` is one such correct process:
+Then
 
-- since `p` is correct, `p` received a valid `POL(v,r)`, and
-- `POL(v,r)` contains a `PREVOTE` message from at least one correct process, and
-- `POL(v,r)` is derived from a timely `POL(v,r*)` with `r* <= r`, and
-- `POL(v,r*)` contains a `PREVOTE` message from at least one correct process, and
-- a correct process considered a proposal for `v` `timely` at round `r*`.
+- there is a valid `POL(v,v.round)` with `v.round <= r` which is a timely proof-of-lock.
+
+The above relation is trivially observed when `r = v.round`, as `POL(v,r)` must
+be a timely proof-of-lock.
+Notice that we cannot have `r < v.round`, as `v.round` is defined as the first
+round at which `v` was proposed.
+
+For `r > v.round` we need to demonstrate that if there is a valid `POL(v,r)`,
+then a timely `POL(v,v.round)` was previously obtained.
+We observe that a condition for observing a `POL(v,r)` is that the proposer of
+round `r` has broadcast a `PROPOSAL` message for `v`.
+As `r > v.round`, we can affirm that `v` was not produced in round `r`.
+Instead, by the protocol operation, `v` was a *valid value* for the proposer of
+round `r`, which means that if the proposer has observed a `POL(v,vr)` with `vr
+< r`.
+The above operation considers a *correct* proposer, but since a `POL(v,r)` was
+produced (by hypothesis) we can affirm that at least one correct process (also)
+observed a `POL(v,vr)`.
+
+> Considering the algorithm in the [arXiv paper][arXiv], `v` was proposed by
+> the proposer `p` of round `round_p` because its `validValue_p` variable was
+> set to `v`.
+> The `PROPOSAL` message broadcast by the proposer, in this case, had `vr > -1`,
+> and it could only be accepted by processes that also observed a `POL(v,vr)`.
+
+Thus, if there is a `POL(v,r)` with `r > v.round`, then there is a valid
+`POL(v,vr)` with `v.round <= vr < r`.
+If `vr = v.round` then `POL(vr,v)` is a timely proof-of-lock and we are done.
+Otherwise, there is another valid `POL(v,vr')` with `v.round <= vr' < vr`,
+and the above reasoning can be recursively applied until we get `vr' = v.round`
+and observe a timely proof-of-lock.
+
+## Temporal analysis
+
+In this section we present invariants that need be observed for ensuring that
+PBTS is both safe and live.
+
+In addition to the variables and system parameters already defined, we use
+`beginRound(p,r)` to represent the time the process `p` reads from its clock
+when it starts round `r` of consensus.
+
+### Safety
+
+The safety of PBTS requires that if a value `v` is decided, then at least one
+correct process `p` considered the associated proposal time `v.time` timely.
```

Comment:
Pardon my ignorance, by why is it not that`2f+1` or even `f+1` processes have to consider it timely?

Reply:
`f+1` processes means at least one correct process, that is the condition assumed here. 

For the sake of safety, we need a correct process to consider `v.time` timely. For the sake of liveness, we need all correct processes (`2f+1` of `f+1`) to consider `v.time` timely.

Reply:
The definition of Time-Validity considers one correct process, it was redefined after some discussion we had.